docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,179 @@
+# Creating the vSphere Windows VM Golden Image
+
+This guide describes the thought process of creating a Windows virtual machine base image (the Golden Image) in vSphere.
+
+In addition, a programmatic approach for building the Windows VM golden image using [Packer](https://github.com/hashicorp/packer)
+is provided [here in the repository](vsphere_ci/README.md).
+
+
+## 1. Select a compatible Windows Server version
+
+Currently, the Windows Machine Config Operator (WMCO) stable version only supports Windows Server version 1909 (build 18363.\*)
+Semi-Annual Channel, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351). 
+Required to get the operating system feature that allows setting the [hybrid OVN Kubernetes networking with a custom 
+VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vsphere)
+
+> Please note that the above patch is not available for Windows Server version 2019.
+
+> Coming soon: Support for Windows Server versions 2004 (build 19041.\*) and 20H2 (build 19042.\*) Semi-Annual Channels.
```

Comment:
This PR adds support for 2004 right? We should remove that from `Coming soon:` and mark `Windows Server versions 2004 (build 19041.\*)` as supported by WMCO.

Reply:
This PR "Adds Windows 2004 template to vSphere", could that be considered as declaring support for WMCO with Windows Server version 2004 (build 19041) ?

Reply:
My thought was that since this doc is vSphere specific we call out WMCO support for 2004 (and not for other cloud providers). Though you are correct, my suggestion may be getting ahead of ourselves.